profile/README.md

```diff
@@ -21,4 +21,4 @@ Flarum is scalable, performant and beautiful appealing to small to enterprise co
 - 🚲 Want to give Flarum a spin? Try the [demo](https://demo.flarum.site) (reset daily) or spin up your own community at [FreeFlarum](https://freeflarum.com).
 - 🧩 Curious which extensions exist? [Extiverse](https://extiverse.com) lists all public and premium Flarum extensions compatible with the latest release.
 - 💻 Wish to see Flarum communities in the wild? Visit [BuiltWithFlarum.com](https://builtwithflarum.com/) or see a global adoption rate at [BuiltWith](https://trends.builtwith.com/cms/Flarum).
-- 💰 Our funds are managed through our [Open Collective](https://opencollective.com/flarum), but we also take donations through [GitHub Sponsors](https://github.com/sponsors/flarum). More information on donating, like how we use the money, can be found at our [Donations page](https://flarum.org/donate).
\ No newline at end of file
+- 💰 Our funds are managed through our [Open Collective](https://opencollective.com/flarum), but we also take donations through [GitHub Sponsors](https://github.com/sponsors/flarum). More information on donating, like how we use the money, can be found at our [Donations page](https://flarum.org/donate).

```

Comment:
This is my editorconfig's doing. Nothing's being changed here except for trimming trailing white spaces.